### ISSUE TITLE
internal compiler error: in elimination_costs_in_insn, at reload1.c:3762

### ISSUE THREAD

**Issue by alexalkis** [bebbo/gcc#155]
git clone https://github.com/paladin-t/my_basic
cd my_basic
m68k-amigaos-gcc -Os -noixemul -o myb shell/main.c core/my_basic.c

if you do without -Os, no error but compiler fails sillently (return code 1 in shell)

**Comment by bebbo** [comment]
https://franke.ms/cex/z/e7qxoe

**Comment by bebbo** [comment]
> if you do without -Os, no error but compiler fails sillently (return code 1 in shell)

there *is* a compile error due to missing `CLOCK_MONOTONIC`.

**Comment by alexalkis** [comment]
is CLOCK_MONOTONIC somehow referenced in source? (I grep'ed and there was nothing found)
Compiler stop reporting internal error after last commit, but there is no executable produced.

Any workarrounds? :)

**Comment by bebbo** [comment]
```
$  m68k-amigaos-gcc -Os -noixemul -o myb shell/main.c core/my_basic.c  -lm
shell/main.c: In function '_ticks':
shell/main.c:1270:2: warning: implicit declaration of function 'clock_gettime' [-Wimplicit-function-declaration]
  clock_gettime(CLOCK_MONOTONIC, &ts);
  ^~~~~~~~~~~~~
shell/main.c:1270:16: error: 'CLOCK_MONOTONIC' undeclared (first use in this function)
  clock_gettime(CLOCK_MONOTONIC, &ts);
...
```

**Comment by bebbo** [comment]
-> `#define CLOCK_MONOTONIC (clockid_t)4`

**Comment by alexalkis** [comment]
Yeah, that works:

m68k-amigaos-gcc -noixemul -DCLOCK_MONOTONIC="(clockid_t)4" -Wno-multichar -Wno-overflow -Wno-unused-result -Os -o myb shell/main.c core/my_basic.c -lm

the thing is, the interpreter doesn't work.  Exe runs, but can't handle (correctly) any bas file.  If you compile for linux it works.
I suspected unicode, I tried to disable it, still no go.   Something else (or even unicode and my test was crap) is messing in libnix?
